code/FinanceManager.Components/Features/FinancialAccounts/Components/InvestmentAccountComponents/InvestmentHoldingCard.razor.css

```
@@ -0,0 +1,85 @@
+    display: flex;
+    flex: 1 1 10rem;
+    flex-direction: column;
+    min-width: 0;
+}
+
+.fm-holding-ticker,
+.fm-holding-exchange,
+.fm-field-label,
+.fm-field-value {
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)